Classes/Item.lua

```diff
@@ -460,13 +460,13 @@ function ItemClass:GetModSpawnWeight(mod, extraTags)
 	local weight = 0
 	if self.base then
 		for i, key in ipairs(mod.weightKey) do
-			if self.base.tags[key] or (extraTags and extraTags[key]) or (self.shaperElderTags and (self.shaper and self.shaperElderTags.shaper == key) or (self.elder and self.shaperElderTags.elder == key) or (self.adjudicator and self.shaperElderTags.adjudicator == key) or (self.basilisk and self.shaperElderTags.basilisk == key) or (self.crusader and self.shaperElderTags.crusader == key) or (self.eyrie and self.shaperElderTags.eyrie == key)) then
+			if self.base.tags[key] or (extraTags and extraTags[key]) or (self.shaperElderTags and ((self.shaper and self.shaperElderTags.shaper == key) or (self.elder and self.shaperElderTags.elder == key) or (self.adjudicator and self.shaperElderTags.adjudicator == key) or (self.basilisk and self.shaperElderTags.basilisk == key) or (self.crusader and self.shaperElderTags.crusader == key) or (self.eyrie and self.shaperElderTags.eyrie == key))) then
```

Comment:
Can we split this giant conditional up by either:
1. Making each conditional a named entity
2. Adding a helper function which creates a bool for each influence)